packages/storybook/stories/Form.stories.tsx

```diff
@@ -0,0 +1,265 @@
+    },
+    title: {
+      type: 'string',
+      value: 'fieldset',
+    },
+    withPanel: {
+      type: 'boolean',
+      value: false,
+    },
+    customStyles: {
+      // type: "object",
+      value: {},
+    },
+  },
+}
+
+export default meta
+
+const FieldsetTemplate: StoryFn<DateInputProps> = (args) => (
+  <Form {...args} onSubmit={(data: unknown) => console.log(data)}>
+    <Select
+      name="dropdown"
+      label="Dropdown"
+      required
+      options={[
+        { value: 'male', label: 'Male' },
+        { value: 'female', label: 'Female' },
+      ]}
+    />
+    <Fieldset label="Personal Details">
+      <Text
+        name="personal_details.firstName"
+        placeholder="John"
+        label="First Name"
+        required
+      />
+      <Text
+        name="personal_details.lastName"
+        placeholder="Doe"
+        label="Last Name"
+        required
+      />
+      <RadioButtonGroup
+        name="personal_details.gender"
+        label="Gender"
+        required
+        radioButtons={[
+          { value: 'male', label: 'Male' },
+          { value: 'female', label: 'Female' },
+        ]}
+      />
+    </Fieldset>
+    <Fieldset label="Contact Details">
+      <Email
+        name="contact_details.email"
+        placeholder="steve@sentosatech.com"
+        label="Email"
+        required
+      />
+      <Text
+        name="contact_details.phone"
+        placeholder="+1 99999999"
+        label="Phone Number"
+        required
+      />
+    </Fieldset>
+    <DateInput
+      name="birthDate"
+      placeholder="07/22/1993"
+      label="Birth Date"
+      required
+    />
+    <NumberInput name="age" placeholder="Age" label="Age" required />
+    <TimeInput name="time" placeholder="Time" label="Time" required />
+    <TextArea
+      name="address"
+      label="Street Address"
+      placeholder="Street Address"
+      required
+    />
+    <RadioButtonGroup
+      name="option"
+      label="Option"
+      required
+      direction="vertical"
+      radioButtons={[
+        { value: 'option 1', label: 'Option 1' },
+        { value: 'option 2', label: 'Option 2' },
+        { value: 'option 3', label: 'Option 3' },
+        { value: 'option 4', label: 'Option 4' },
+      ]}
+    />
+    <CheckBox name="agree" label="Agree to Terms and Conditions" required />
+    <Button
+      parentButtonProps={{ type: 'submit' }}
+      text="Submit"
+      primary
+      rounded
+    />
+  </Form>
+)
+
+export const Default = FieldsetTemplate.bind({})
+Default.args = {
+  title: 'Form Title',
+  label: 'Default Fieldset Label',
+  withPanel: true,
+}
+
+export const Debug = FieldsetTemplate.bind({})
+Debug.args = {
+  title: 'Form With Debug Enabled',
+  label: 'Default Fieldset Label',
+  withPanel: true,
+  debug: true,
+}
+
+export const WithoutPanel = FieldsetTemplate.bind({})
+WithoutPanel.args = {
+  title: 'Form Title',
+  withPanel: false,
+}
```

Comment:
@ianmarquez we might be missing here a default style for the background.

Reply:
this is intended as the panel is whats giving the background color. if none it would inherit its parents background.